include/pistache/tcp.h

```diff
@@ -25,9 +25,7 @@ enum class Options : uint64_t {
     Linger               = NoDelay << 1,
     FastOpen             = Linger << 1,
     QuickAck             = FastOpen << 1,
-    ReuseAddr            = QuickAck << 1,
-    ReverseLookup        = ReuseAddr << 1,
```

Comment:
You removed ReverseLookup (unintentional)?

Reply:
Actually it was intentional because ReverseLookup isn't being used anywhere.